crates/pipeline-types/src/config.rs

```diff
@@ -266,11 +266,11 @@ pub struct ConnectorConfig {
 }
 
 fn default_max_buffer_time_millis() -> usize {
-    usize::MAX
+    i64::MAX as usize
```

Comment:
this is a funny one, it was valid in yaml, but in (serde) json the max integer seems to be i64::MAX

Reply:
I general I think we should not have usize anywhere in our configuration types